README.md

```diff
@@ -73,15 +72,18 @@ documentation](https://docs.rubocop.org/rubocop/configuration.html#inheriting-co
 
 * Avoid monkeypatching.
 
-* Keep the code simple.
```

Comment:
Removed the following rules since they are not about the "style in which you write Ruby" and more about general guidelines when programming. These are all very much implicit and I don't see the value of mentioning any of them:

* Keep the code simple.
* Do the right amount of design, not too little but also not too much.
* Avoid bugs.
* Be consistent.
* Use common sense.